### ISSUE TITLE
What is the final performance?

### ISSUE THREAD

**Issue by zzw1123** [zrl4836/ACFNet#2]
Hi,
Thanks for your contribution! I wanna know what's the final mIoU of your training code? Is it the same as the paper?
Meanwhile, I notice that the learning rate you set is 1e-2 while it is 1e-3 in the original paper. Have you tried 1e-3?
Thank you again! All the best!

**Comment by zrl4836** [comment]
ennn..I don't have enough GPUs to train the model.you can view issue#1,I am also waiting for his result.

**Comment by zzw1123** [comment]
Alright, good luck then.
